ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/LedgerPeers.hs

```diff
@@ -95,7 +98,7 @@ newtype ArbitrarySlotNo =
 -- of the tests we run.
 instance Arbitrary ArbitrarySlotNo where
     arbitrary =
-      ArbitrarySlotNo . fromInteger <$> arbitrary
+      ArbitrarySlotNo . SlotNo <$> arbitrarySizedBoundedIntegral
```

Comment:
:+1: I would just use `arbitrary`, since `Wor64` is using `arbitrarySizedBoundedIntegral` underneath.